string NAME=took_off
bool ERASE_WHEN_FALSE=0

ObjectDroneID drone_id

bool truth_value
